src/App.jsx

```diff
@@ -17,56 +17,57 @@ export const goods = [
 ];
 
 export const App = () => {
-  const [isActive, selectedGood] = useState('Jam');
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
   const stateOfGood = (good) => {
-    const check = isActive === good ? selectedGood('') : selectedGood(good);
+    const check = selectedGood === good
+      ? setSelectedGood('') : setSelectedGood(good);
 
     return check;
   };
 
-  const checkGoods = isActive ? `${isActive} is selected` : 'No goods selected';
+  const checkGoods = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
 
   return (
     <main className="section container">
       <h1 className={
-        cn('title', { 'title is-flex is-align-items-center': isActive })}
+        cn('title', { 'title is-flex is-align-items-center': selectedGood })}
       >
         {checkGoods}
 
-        {isActive && (
+        {selectedGood && (
           <button
             data-cy="ClearButton"
             type="button"
             className="delete ml-3"
-            onClick={() => (selectedGood(current => ''))}
+            onClick={() => (setSelectedGood(current => ''))}
```

Comment:
You don't need callback for setting state here. You can use just value. We need a callback if our new state depends on the previous one